bedrock/firefox/urls.py

```diff
@@ -31,8 +31,10 @@
     url(r'^firefox/(?:%s/)?(?:%s/)?all/$' % (platform_re, channel_re),
         views.all_downloads, name='firefox.all'),
     page('firefox/accounts', 'firefox/accounts.html'),
-    page('firefox/channel', 'firefox/channel.html'),
-    redirect('^firefox/channel/android/$', 'firefox.channel', locale_prefix=False),
+    redirect('^firefox/channel/$', 'firefox.channel.desktop', locale_prefix=False),
```

Comment:
Good question, I'm not sure why the redirect was here, probably some legacy thing. I'll see if I can move the redirect to `redirects.py`